.travis.yml

```diff
@@ -12,5 +12,6 @@ before_install:
 install:
   - "pip install xcffib"
   - "pip install ."
+  - "if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then python cairocffi/ffi_build.py; fi"
```

Comment:
My point is that end users should be able to use cairocffi without running a build script manually, and CI should make sure that this works.

Shouldn’t `cairocffi._ffi` and `._ffi` be the same thing?

I think that `py.test` by default looks for files in the current directory, which is not those that `pip install` installed. I’ve had similar issues with tox. Try `script: "py.test --pyargs cairocffi"` ?